core/src/main/java/discord4j/core/object/entity/Guild.java

```diff
@@ -544,15 +544,14 @@ public Mono<GuildEmoji> getGuildEmojiById(final Snowflake id, EntityRetrievalStr
     }
 
     /**
-     * Gets the enabled guild features.
+     * Gets the enabled features of this {@link Guild}.
      * <br>
-     * You can see the available
-     * <a href="https://discord.com/developers/docs/resources/guild#guild-object-guild-features">guild features</a>
+     * Raw data features are still available with {@link #getData()} using {@link GuildData#features()}.
      *
-     * @return The enabled guild features.
+     * @return A {@code EnumSet} with the enabled guild features.
      */
-    public Set<String> getFeatures() {
-        return new HashSet<>(data.features());
+    public EnumSet<FeatureSet> getFeatures() {
```

Comment:
I think that you can leave this method, but mark it as deprecated, and create a new `getGuildFeatures` method.